pbm/archive/conv.go

```diff
@@ -0,0 +1,133 @@
+package archive
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"runtime"
+	"slices"
+	"strings"
+
+	mtArchive "github.com/mongodb/mongo-tools/common/archive"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/percona/percona-backup-mongodb/pbm/storage"
+)
+
+func GenerateV1FromV2(stg storage.Storage, bcp, rs string) error {
```

Comment:
Do we have the Jira ticket to align backup and restore about the metadata format? So currently we use V2->V1 conversion. In future we should make backup to export directly to V1, or we should make restore to work with V2 format directly.

Reply:
v1 or v2 - does not matter. converting to v1 was done as quick solution. it is pbm format. when PBM creates archive for mongorestore, it uses data from the v1 but not format directly. PBM can makes the archive from v2. just some changes are needed in another place.